src/content/en/showcase/2017/matterport2.md

```diff
@@ -0,0 +1,122 @@
+uncomfortable flickering:
+
+<a href="https://www.matterport.com/virtual-reality/webvr/"><img
+src="/web/showcase/2017/images/matterport/timeline.png" class="attemptfg-right"></a>
+
+Until `vrdisplayactivate` fires earlier, using `onclick` and a button provides users with the best
+experience for now.
+
+## Performance
+
+We knew performance would have to be rock solid for the experience to work well, so we started with
+the assumption that we would only implement a bare bones experience: no bells and whistles, just
+simple navigation inside a space.
+
+We quickly made a prototype with <a
+href="https://github.com/borismus/webvr-boilerplate">webvr-boilerplate</a> and a stripped down
+version of our loading and rendering code. In our first run we only got about 10-20 FPS: In our
+normal WebGL app, that would feel bad, but in VR it’s downright nauseating.  As we often do, we
+fired up the Chrome Developer Tools, and connected it to a Pixel phone. We got four major
+takeaways:
+
+- Know your rendering code!
+    - Webvr-boilerplate rendered the screen a total of four times, two for each eye. Removing one
+      extraneous line of code quickly halved our rendering time! This has since been fixed in the
+      newest version of webvr-boilerplate, but we suspect that this particular bug has caused a lot
+      of developers to get the wrong impression of webvr performance.
+- Profile on the actual device you are building for, not your developer computer
+    - Mobile devices and browsers have very different performance characteristics from your Mac;
+      make sure you’re optimizing the real bottleneck.
+- You don’t need all those pixels
+    - Be aware how many pixels you are rendering. In THREE.js, this is set with
+      WebGLRenderer.setPixelRatio(). You might not need the full resolution, in our case we simply
+      set it to 1.0, as higher numbers did not give visibly better visuals.
+- Be smart about the raycasting
+    - Our meshes are generally around 50,000 faces, which can be a bit expensive to raycast in. We
+      split our scene into smaller cubes and raycast into that, but you can also consider an
+      Octree.
+
+After a couple of rounds of improvements, we are now seeing super smooth experience on both the
+Pixel and the Pixel XL.
+
+## Living on the Bleeding Edge
+
+When you’re making something with a browser feature that is this new, you’re bound to see a lot of
+changes. In the WebVR specification, there was a mention of a security feature for iframes, which
+embedders would have to add to their embed code to allow WebVR. This is important to us since our
+WebGL player is embedded in iframes all around the web. The new property looked something like
+this:
+
+    <iframe src="..." allowfullscreen allowvr>
+
+We started changing our documentation and embed codes that we hand out to customers, but wanted to
+check with the Chrome team that this was really needed. **It wasn’t!** The Chrome team quickly
```

Comment:
What I'm about to describe is a judgement call. Contrary to popular belief it is occasionally permissible to have a single sentence paragraph. It's done for emphasis. I would remove the bolding and exclamation point and make this it's own paragraph, then add the next sentence to the following paragraph.